lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -121,18 +121,27 @@ tokenize(String, Line, Column, Opts) ->
 
   Existing = case lists:keyfind(existing_atoms_only, 1, Opts) of
     {existing_atoms_only, true} -> true;
+    {existing_atoms_only, false} -> false;
```

Comment:
@uohzxela for @michalmuskala's suggestion to work, you will need to come with a different var name rather than Bool for each usage. My suggestion is to use the name of the outer option with the Bool suffix. So `ExistingAtomsBool`, `PreserveCommentsBool` and so on.